src/DetailsView/components/should-show-report-export-button.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { VisualizationType } from 'common/types/visualization-type';
+import { CommandBarProps } from 'DetailsView/components/details-view-command-bar';
+
+export type ShouldShowReportExportButton = (props: CommandBarProps) => boolean;
+
+export function shouldShowReportExportButtonForAssessment(props: CommandBarProps): boolean {
+    return true;
+}
+
+export function shouldShowReportExportButtonForFastpass(props: CommandBarProps): boolean {
+    const scanResult = props.visualizationScanResultData.issues.scanResult;
```

Comment:
Would be good to make the change here to not use the scan result store but rather the unified store.